test/unit/test_sap_adt_connection.py

```
@@ -14,6 +14,52 @@
+    def test_with_content_type(self, fake_new_adt_error_from_xml):
+        client = Mock()
+        req = Mock()
+
+        res = Mock()
+        res.status_code = 500
+        res.headers = {'content-type': 'application/xml'}
+        res.text = 'standard ADT error'
+
+        fake_new_adt_error_from_xml.return_value = sap.adt.errors.ADTError('com.sap.adt', 'ADHOC', 'Parsed ADT error')
+
+        with self.assertRaises(sap.adt.errors.ADTError) as cm:
```

Flake8 (7.3.0)

[error] local variable 'cm' is assigned to but never used

(F841)